rootfs/etc/nginx/lua/test/monitor_test.lua

```diff
@@ -54,13 +51,10 @@ describe("Monitor", function()
             local expected_json_stats = {
                 host = "testshop.com",
                 status = "200",
-                remoteAddr = "10.10.10.10",
-                realIpAddr = "5.5.5.5",
-                remoteUser = "admin",
                 bytesSent = 150.0,
                 protocol = "HTTP",
                 method = "GET",
-                uri = "/admin",
+                path = "/admin",
```

Comment:
Why not `location_path`?

Reply:
```
location_path -> from nginx (variable)
path               -> go side
```

Reply:
Ah right, `nginx_environment` vs. `expected_json_stats`